aFRR-Service/DtoConverters/AssetDTO.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace aFRRService.DTOs;
```

Comment:
Double check the namespaces of both files

Reply:
Approved with: Commit [8e552fa](https://github.com/diversiwaver/aFRR-NET/pull/131/commits/8e552fae5090b6ce721cb81542ba109e9432eceb)